filters/ratelimit/failure_mode.go

```diff
@@ -0,0 +1,76 @@
+package ratelimit
+
+import (
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/routing"
+)
+
+type FailureModeSpec struct{}
```

Comment:
Should it be named after filter, i.e. `RatelimitFailClosedSpec`?

Reply:
done